### ISSUE TITLE
[Discussion] Evaluate Vitest for unit tests

### ISSUE THREAD

**Issue by Cahllagerfeld** [EddieHubCommunity/good-first-issue-finder#309]
### Description

Do we need to set up Unit tests?

If we want, we should definitely use [Vitest](https://vitest.dev/), as it best integrates with Sveltekit and other Vite apps.

**Comment by shubhamjena18** [comment]
@Cahllagerfeld  , yes Vitest is a great option for unit tests. But there is another option called [Playwright](https://playwright.dev/). Can we have a comparision and choose the best one ?

**Comment by Cahllagerfeld** [comment]
Youre right Playwright is out there and we already use it in the project. I think these two setups solve different problems though:
- Vitest is a framework for running unit tests, while
- Playwright is for running end-to-end tests

We should evaluate Vitest for unit tests while keeping Playwright for end-to-end tests.

**Comment by shubhamjena18** [comment]
ok great. Can I work on implementing vitest in this project.

**Comment by Cahllagerfeld** [comment]
sure 👍

**Comment by shubhamjena18** [comment]
@Cahllagerfeld  I am stuck in some personal issues , so I won't be able to work on this task for some time .
 Can you re assign the task to someone who is interested .
